.vscode/launch.json

```diff
@@ -2,11 +2,20 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "type": "pwa-chrome",
```

Comment:
Any reason not to keep this aswell as accessing the local Chrome conveniently? [Haven't testing this PR yet, so this may be non-sensical]

Reply:
Actually, I tried very hard to keep it. Why I didn't keep it:

- It causes VS Code to enter in debug mode, and there is nothing to debug from Chrome side.
  - Therefore, VS Code would be attached to two debug sessions at the same time. Although it works, it clutters the UI without being necessary.
  - Frequently I found the Chrome debugger forwarding ports randomly and I could not find a way to prevent it from doing so. It was polluting the _Ports_ tab, every time I ran F5.
  - Frequently it was pausing in exceptions (random JS exceptions without any hints), and I could not make it stop doing that.
  
Next, I thought to simply open the browser automatically after attaching the Python debugger. There are many ways of doing so, like running `sensible-browser http://localhost:8123` in the command line. But VSCode Remote seems to have a bug that does not recognize `localhost:8123` as the `hass:8123` forwarded port, and auto-forward from `localhost:8123` (which would be the dev container -- localhost:8123 does not work there) to the next available port, `8124`, and therefore does not open.

I plan to open an issue in the VS Code repository, but I didn't find the energy yet.

So, until the problem above gets fixed (or a work around is found), I suggest to just open localhost:8123 manually or with the help of the _Ports_ tab.

Reply:
OK -- got it. That's a shame, but no big deal.

Reply:
FYI:
- https://github.com/microsoft/vscode-remote-release/issues/6895